includes/class-wc-dintero-hp-ajax.php

```diff
@@ -121,7 +121,7 @@ public static function add_ajax_events() {
 			'express_pay',
 			'dhp_update_ord',
 			'dhp_update_ord_emded',
-			'dhp_update_ship',
+			'dhp_shipping_options',
```

Comment:
@chaikovskyia Can you make `dhp_update_ship` respond to the calls as well, so that sessions created before an update to this version can still get their shipping_options updated?